test/e2e/singlecluster/statefulset_test.go

```diff
@@ -126,11 +126,11 @@ var _ = ginkgo.Describe("Stateful set integration", func() {
 				conflictingWorkload := &kueue.Workload{}
 				gomega.Expect(k8sClient.Get(ctx, conflictingWlLookupKey, conflictingWorkload)).To(gomega.Succeed())
 				gomega.Expect(createdWorkload.Name).ToNot(gomega.Equal(conflictingWorkload.Name))
-				util.ExpectObjectToBeDeleted(ctx, k8sClient, conflictingStatefulSet, true)
```

Comment:
if it works but takes more time can we have a new function `util.ExpectObjectToBeDeletedWithTimeout` and pass the LongTimeout instead

Reply:
Done